build.gradle

```diff
@@ -12,14 +12,15 @@ compileJava {
 
 repositories {
     mavenLocal()
+    mavenCentral()
     maven {
-        url "https://oss.jfrog.org/artifactory/libs-snapshot"
+        url "https://releases.jfrog.io/artifactory/oss-snapshots"
```

Comment:
Is this still relevant? Are e going to have an oss-snapshots in releases.jfrog.io?